resources/templates/mustache/production/govuk-template.mustache.html

```diff
@@ -196,15 +196,15 @@
                                             {{#cyUrl}}
                                                 {{#isWelsh}}
                                                     <li>
-                                                        <a id="switchToEnglish" lang="en" href="{{enUrl}}" data-journey-click="link - click:lang-select:English"><small><strong class="bold">English</strong></small></a>
-                                                        <span class="faded-text--small"><strong class="bold">| Cymraeg</strong></span>
+                                                        <a id="switchToEnglish" lang="en" href="{{enUrl}}" data-journey-click="link - click:lang-select:English" aria-label="Select English">English</a>
+                                                        <span class="faded-text--small bold"  >| Cymraeg</span>
                                                     </li>
                                                 {{/isWelsh}}
 
                                                 {{^isWelsh}}
                                                     <li>
-                                                        <span class="faded-text--small"><strong class="bold">English |</strong></span>
-                                                        <a id="switchToWelsh" lang="cy" href="{{cyUrl}}" data-journey-click="link - click:lang-select:Cymraeg"><small><strong class="bold">Cymraeg</strong></small></a>
+                                                        <span class="faded-text--small bold" aria-hidden="true">English |</span>
+                                                        <a id="switchToWelsh" lang="cy" href="{{cyUrl}}" data-journey-click="link - click:lang-select:Cymraeg" aria-label="Dewiswch Cymraeg">Cymraeg</a>
```

Comment:
You dont have the quotes this time

Reply:
They have been removed from frontend-template as well but it was a separate pr that fixed failing tests and removed quotes. sorry

Reply:
Ahh, I didnt check master branch, I only checked that PR. My bad.